util/build_configs/cray-internal/README.md

```diff
@@ -57,7 +57,6 @@ new subdir.
   are Cray-specific implementations of the general build_configs/setenv pattern:
   - setenv-xc-x86_64:  Chapel Cray-XC module for x86_64
```

Comment:
No hyphen in "Cray XC".  This is actually a general flaw in `$CHPL_HOME/util` which I should have caught in the beginning.  It could be fixed separately.

Reply:
In this case I think `Cray-XC` could be right because it's talking about our module, and not the platform as a whole. Either way I'm not really sure these are worth addressing.